src/common/POLFields/FieldHolding/FieldHolding.js

```diff
@@ -0,0 +1,155 @@
+import React, { useMemo, useCallback, useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { useFormState } from 'react-final-form';
+import { get } from 'lodash';
+
+import { LocationLookup } from '@folio/stripes/smart-components';
+import {
+  IconButton,
+  KeyValue,
+  Loading,
+  TextField,
+} from '@folio/stripes/components';
+import {
+  FieldSelectionFinal,
+  validateRequired,
+} from '@folio/stripes-acq-components';
+
+import { useHoldings } from './useHoldings';
+
+const FieldHolding = ({
+  instanceId,
+  isDisabled,
+  labelId,
+  locationFieldName,
+  locationLabelId,
+  locationsForDict,
+  name,
+  onChange,
+  required,
+}) => {
+  const { isHoldingsLoading, holdings } = useHoldings(instanceId);
+  const [selectedLocation, setSelectedLocation] = useState();
+  const { values } = useFormState();
+
+  const locationsMap = useMemo(() => (locationsForDict.reduce((acc, loc) => {
+    acc[loc.id] = loc;
+
+    return acc;
+  }, {})), [locationsForDict]);
+
+  useEffect(() => {
+    if (!get(values, name)) {
```

Comment:
yep, get is not needed here

Reply:
I use `get` because `name` here looks like `locations[0].holdingId` inside repeatable field

Reply:
I don't think I'm following...
Could you elaborate&

Reply:
I mean that Field `locations`  is an array. Prop `name` here is string [`'locations[0].holdingId'`](https://github.com/folio-org/ui-orders/pull/1140/files#diff-7c1b89acfe198286a0ee92f000ebfd6fd5593028f658e10666855d77aa7bc8a1R62). And `lodash/get` provides a possibility to get the value from this kind of string. So `values['locations[0].holdingId']` will alwayes be undefined)

Reply:
Ah, thanks, now I get it